vpr/src/base/SetupVPR.cpp

```diff
@@ -503,8 +503,10 @@ static void SetupPlacerOpts(const t_options& Options, t_placer_opts* PlacerOpts)
 
     PlacerOpts->place_algorithm = Options.PlaceAlgorithm;
 
+    PlacerOpts->constraints_file = Options.constraints_file;
     PlacerOpts->pad_loc_file = Options.pad_loc_file;
     PlacerOpts->pad_loc_type = Options.pad_loc_type;
+    PlacerOpts->block_loc_type = Options.block_loc_type;
```

Comment:
block_loc_type isn't a great name; not obvious it goes with the constraints file.
But probably better still to simply have a NULL or empty constraints_file name mean no location constraints.

Reply:
I think the best thing is to delete block_loc_type entirely and use a NULL constraints file name to mean no constraints. Comment it when you do the test for a NULL constraints file (means none specified).